client/platform/desktop/backend/native/common.ts

```diff
@@ -127,7 +132,19 @@ async function loadJsonMetadata(metaAbsPath: string): Promise<JsonMeta> {
  */
 async function loadJsonTracks(tracksAbsPath: string): Promise<MultiTrackRecord> {
   const rawBuffer = await fs.readFile(tracksAbsPath, 'utf-8');
-  const annotationData = JSON.parse(rawBuffer) as MultiTrackRecord;
+  if (rawBuffer.length < 5) {
+    /**
+     * 5 is somewhat arbitrary, and accounts for newline chars
+     * This cannot possibly be a valid file.  Return empty
+     */
+    return {};
```

Comment:
I'm not exactly sure how I feel about this.  A file that is truly empty or a file that is just `{}` feels different than a file that contains `12345` or `null` (it shouldn't have it but who knows) or something else that falls within that 5 limit but cannot be parsed.  Feels like it should error then.  Should we have explicit cases where it works:  Missing file, blank file or empty object file?

Reply:
I'm guarding against newline characters that make the file look empty.

I think you're right that this is too permissive.  Perhaps we should strip newlines from the input before parsing, then safely handle the 0 length case, and otherwise throw an error.

Reply:
Overthinking it. Need to handle the empty file case, but otherwise, I think a parser error is appropriate.